### ISSUE TITLE
Support for Ethereum 2.x

### ISSUE THREAD

**Issue by PabloLefort** [pipermerriam/eth-testrpc#99]
There is something that prevents support for `ethereum==2.x` ?

Thanks in advance :rocket:

**Comment by johannes-schuetze** [comment]
I'm trying to use python3.6 with ethereum-2.x.x (i need ethereum.tools) and I am using Populus ("testrpc" chain) for testing. This causes a dependency on eth_testrpc library which has the dependency on ethereum-1.6.1 (pyethereum).

**Comment by PabloLefort** [comment]
Yes @johannes-schuetze i was facing something similar when using eth_testrpc.
If you use Docker, take a look at `docker pull lefortpablo/ethereum-test`
